ocamldoc/Makefile

```diff
@@ -180,13 +180,25 @@ LIBCMOFILES = $(CMOFILES)
 LIBCMXFILES = $(LIBCMOFILES:.cmo=.cmx)
 LIBCMIFILES = $(LIBCMOFILES:.cmo=.cmi)
 
+P :=
+include ../stdlib/StdlibModules
+
+STDLIB_MODULES := pervasives $(filter-out stdlib,$(STDLIB_MODULES))
+PARSING_MLIS := $(wildcard ../parsing/*.mli)
+
 STDLIB_MLIS=\
-  ../stdlib/*.mli \
-  ../parsing/*.mli \
-  ../otherlibs/$(UNIXLIB)/unix.mli \
-  ../otherlibs/str/str.mli \
-  ../otherlibs/bigarray/bigarray.mli \
-  ../otherlibs/num/num.mli
+  $(STDLIB_MODULES:%=stdlib_non_prefixed/%.mli) \
+  $(PARSING_MLIS:../parsing/%.mli=stdlib_non_prefixed/%.mli) \
+  stdlib_non_prefixed/warnings.mli \
+  stdlib_non_prefixed/unix.mli \
+  stdlib_non_prefixed/str.mli \
+  stdlib_non_prefixed/bigarray.mli \
+  stdlib_non_prefixed/num.mli \
+  stdlib_non_prefixed/nat.mli \
+  stdlib_non_prefixed/big_int.mli \
+  stdlib_non_prefixed/ratio.mli
```

Comment:
I agree that this is not obvious but ocamldoc' makefile is totally unrelated with the generation of the stdlib documentation for the manual that is done in manual/manual/Makefile.